### PR TITLE
feat: add clustering key in create/describe collection

### PR DESCRIPTION
**Pull request by wayblink**
#28410
/kind feature

### CODE REVIEW

File: internal/proxy/task.go [hunk 1]
```diff
@@ -207,6 +207,33 @@ func (t *createCollectionTask) validatePartitionKey() error {
...
+			if idx != -1 {
+				return fmt.Errorf("there are more than one clustering key, field name = %s, %s", t.schema.Fields[idx].Name, field.Name)
+			}
+
+			if field.GetIsPrimaryKey() {
+				return errors.New("the clustering key field must not be primary field")
+			}
+
+			if field.GetIsPartitionKey() {
+				return errors.New("the clustering key field must not be partition field")
```
Comment (czs007) [inline internal/proxy/task.go:223]: need to refine error message

Comment (wayblink) [reply internal/proxy/task.go]: updated

**Comment by czs007** [comment]
@wayblink need rebase to resolve UT for go

**Comment by wayblink** [comment]
rerun ut

**Comment by wayblink** [comment]
rerun ut

**Pull request merged**